README.md

```diff
@@ -56,6 +56,25 @@ cargo release 0.1.0
 
 That's gonna do a whole bunch of stuff you might not have expected (but I left off the --execute flag from `cargo-release` so you won't actually break anything if you really did just copy paste that 😇). Read the rest of the docs to know what it does!
 
+Or if you don't want to use cargo-release, you can emulate its behaviour manually:
```

Comment:
can we show it this way first and then be like- hey cargo-release does some of this if you'd like to use it!